content/news/march-2016-tournament-annoucment.md

```diff
@@ -0,0 +1,11 @@
+[March 2016 OpenRA Tournament](..\images\news\march-2016-tournament.jpg)  If you are interested in some heated tag-team battles in OpenRA [Register Your Team Here] (http://www.sleipnirstuff.com/forum/viewtopic.php?f=82&t=19414)!
+
+It Will Run From: March 5th-6th and 11th-12th 2016!
+Game Progression will be as follows: 
+Each player will first face off against an opponent in a 1v1 no-holds-barred fight to the death! 
+Then the teams will join forces in an attempt to wipe their adversary from the face of the planet in a 2v2 Superpower Scrap!!!
```

Comment:
I thought that would have been a bit much as it's listed in the official rules...but will put it in.